lib/presenter.rb

```diff
@@ -5,20 +5,13 @@ def initialize(grid)
     @grid = grid
   end
 
-  def print
-    STDOUT.print "\n"
-    grid.height.times do |y|
-      grid.width.times do |x|
-        print_cell(grid.cells[y][x])
-      end
-      STDOUT.print "\n"
-    end
+  def print_grid
+    print grid.cells.map { |row| row_string(row) }.join("\n")
   end
 
   private
 
-  def print_cell(cell)
-    val = cell.alive? ? '• ' : '  '
```

Comment:
I think the point of this presenter was to take the view logic out of the cell model. But fuck it dawg, it's Friday night.